.github/semantic.yml

```diff
@@ -0,0 +1,5 @@
+# Always validate the PR title AND all the commits
+titleAndCommits: true
+# Allows use of Merge commits (eg on github: "Merge branch 'master' into feature/ride-unicorns")
+# this is only relevant when using commitsOnly: true (or titleAndCommits: true)
+allowMergeCommits: true
```

Comment:
let's set this to `false`, we only use rebase or squash and merge

Reply:
- [x] Fixed in 612db406731da7c58de5bed89c0a931f1de52355